pkg/authorization/registry/clusterpolicy/etcd/etcd.go

```diff
@@ -21,11 +22,12 @@ type REST struct {
 }
 
 // NewStorage returns a RESTStorage object that will work against nodes.
-func NewStorage(s storage.Interface) *REST {
+func NewStorage(optsGetter restoptions.Getter) *REST {
```

Comment:
RESTOption isn't an interface. This Getter allows the storage to obtain a RESTOption struct